config/routes.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  devise_for :users
+  
+  resources :splash, only: [:index, :logout]
+  resources :group, only: [:index, :new, :create, :show] do
+    resources :payment
```

Comment:
- [ ] Great work on fixing the routes for the group. But now, I see you didn't implement the same for the payment. When I check the payment controller, the index route is not used and the destroy route is also not used. Also the edit and update routes. Kindly ensure that only the routes that are required in your application are the only ones generated.